kraken/plugins/network/cerberus.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import requests
+import sys
+import json
+
+
+# Get cerberus status
```

Comment:
This comment can be converted to be python doc string for the function.

Reply:
Added in [cd49b7e](https://github.com/redhat-chaos/krkn/pull/299/commits/cd49b7ebcd0fe4fc12ee20e98e3fdf7db7edcc27)